Mail2Bug/Email/IIncomingEmailMessage.cs

```diff
@@ -15,7 +15,13 @@ public interface IIncomingEmailMessage
         string Subject { get; }
         string RawBody { get; }
         string PlainTextBody { get; }
+
```

Comment:
We should avoid adding this as a distinct method in IIncomingEmailMessage, since this is Exchange specific.
Instead, we should have the two methods in EWSIncomingMailMessage, and make ConversationIndex call one or the other based on the config